### ISSUE TITLE
Copy for supported integrations

### ISSUE THREAD

**Issue by StuartFeldt** [StuartFeldt/Hashtags#23]

**Comment by a1exis** [comment]
I don't know where the right number is to leave this comment, but i'm throwing a dart and hoping this relates somehow. I don't think we should support pinterest. People just don't use it in the same way they use other social mediums--rarely in real-time.  If anything, we should support facebook (if at all possible) instead.  Twitter and Instagram are key, though.

**Comment by StuartFeldt** [comment]
Right, I don't think people will use it in realtime, but it may be nice use it to add pins into the rotation.  Like if you are using it for a wedding, you pin stuff that you want to be seen.  

Facebook is a bit tricky, as most people's posts aren't public.  But, if we authenticate the owner's facebook account, we may be able to read from their timeline.

Also, Pinterest issue here: #43

**Comment by a1exis** [comment]
Assuming this is for the link from Supported Social Platforms. (I like the look of it, btw) Let me know what your final verdict is on what mediums will be supported, and what 'options' the person has.  The pop-up windows with 'more info' is something i'll need guidance with, because i won't understand the innerworkings exactly
